src/marketing/yotpo-reviews-intro.md

```diff
@@ -16,3 +16,17 @@ See the [Yotpo website](https://www.yotpo.com/platform/visual-marketing/) for mo
 ## Integrating Your Magento Store with your Yotpo Account
 
 With Magento v2.3.3+, integrating your Magento store is a quick, simple process and gives you access to the content and data customizations available in your Yotpo account. See [Integrate your Magento Store with your Yotpo Account]({% link marketing/yotpo-connect.md %}).
+
+## Changes In the Latest Release
+
+This release contains changes to correct a variety of issues and to update documentation and Yotpo-specific links.
+
+- The **Yotpo Reviews** and **Yotpo Analytics** links in the Admin > Reports section have been updated to redirect to the connected Yotpo account information.
+
+- Corrected issues related to the syncing of invalid orders. Invalid orders are now listed in the log, and additional sync are not attempted.
+
+- Corrected an issue preventing orders from syncing correctly.
+
+- Updated _Yotpo Setup Guide_ links in the Admin.
+
+- Corrected an issue preventing Yotpo settings from displaying in the Admin after setup, when `Single-Store` mode is selected.
```

Comment:
```suggestion
- Corrected an issue preventing Yotpo settings from displaying in the Admin after setup when `Single-Store` mode is selected.
```

Reply:
made change directly in file. unable to commit from suggestion